Sourcery/Parsing/FileParser.swift

```diff
@@ -144,16 +144,16 @@ final class FileParser {
             case .enumelement:
                 return parseEnumCase(source)
             case .varInstance:
-                return parseVariable(source, containedInProtocol: containingIn is SourceryProtocol)
+                return parseVariable(source, containedIn: containingIn as? Type)
```

Comment:
I like `definedIn` and I've replaced the confused part in `FileParser` in d7da21b.